ClientTests/RelativeDatesTests.swift

```diff
@@ -10,38 +10,31 @@ class RelativeDatesTests: XCTestCase {
         let dateOrig = Date()
         var date = Date(timeInterval: 0, since: dateOrig)
         
-        XCTAssertTrue(date.toRelativeTimeString() == "just now")
+        XCTAssertEqual(date.toRelativeTimeString(), "just now")
         
-        date = Date(timeInterval: 0, since: dateOrig)
-        date = date.addingTimeInterval(-10)
-        XCTAssertTrue(date.toRelativeTimeString() == "just now")
-        
-        date = Date(timeInterval: 0, since: dateOrig)
-        date = date.addingTimeInterval(-60)
-        XCTAssertTrue(date.toRelativeTimeString() == ("today at " + DateFormatter.localizedString(from: date, dateStyle: DateFormatter.Style.none, timeStyle: DateFormatter.Style.short)))
-        
-        date = Date(timeInterval: 0, since: dateOrig)
-        date = date.addingTimeInterval(-60 * 60 * 24)
-        XCTAssertTrue(date.toRelativeTimeString() == "yesterday")
-        
-        date = Date(timeInterval: 0, since: dateOrig)
-        date = date.addingTimeInterval(-60 * 60 * 24 * 2)
-        XCTAssertTrue(date.toRelativeTimeString() == "this week")
-        
-        date = Date(timeInterval: 0, since: dateOrig)
-        date = date.addingTimeInterval(-60 * 60 * 24 * 7)
-        XCTAssertTrue(date.toRelativeTimeString() == "more than a week ago")
+        date = Date(timeInterval: -10, since: dateOrig)
+        XCTAssertEqual(date.toRelativeTimeString(), "just now")
+
+        date = Date(timeInterval: -60, since: dateOrig)
+        XCTAssertEqual(date.toRelativeTimeString(), ("today at " + DateFormatter.localizedString(from: date, dateStyle: DateFormatter.Style.none, timeStyle: DateFormatter.Style.short)))
+
+        let calendar = Calendar.autoupdatingCurrent
+        date = calendar.date(byAdding: .day, value: -1, to: dateOrig)!
+        XCTAssertEqual(date.toRelativeTimeString(), "yesterday")
+
+        date = calendar.date(byAdding: .day, value: -2, to: dateOrig)!
+        XCTAssertEqual(date.toRelativeTimeString(), "this week")
+
+        date = calendar.date(byAdding: .day, value: -7, to: dateOrig)!
+        XCTAssertEqual(date.toRelativeTimeString(), "more than a week ago")
         
-        date = Date(timeInterval: 0, since: dateOrig)
-        date = date.addingTimeInterval(-60 * 60 * 24 * 7 * 5)
-        XCTAssertTrue(date.toRelativeTimeString() == "more than a month ago")
+        date = calendar.date(byAdding: .day, value: -7 * 5, to: dateOrig)!
+        XCTAssertEqual(date.toRelativeTimeString(), "more than a month ago")
         
-        date = Date(timeInterval: 0, since: dateOrig)
-        date = date.addingTimeInterval(-60 * 60 * 24 * 7 * 5 * 2)
-        XCTAssertTrue(date.toRelativeTimeString() == DateFormatter.localizedString(from: date, dateStyle: DateFormatter.Style.short, timeStyle: DateFormatter.Style.short))
+        date = Date(timeInterval: -60 * 60 * 24 * 7 * 5 * 2, since: dateOrig)
```

Comment:
Could be lack of coffee -and this comment is not related to your changes-  but I am having trouble seeing what this test does. I like that `calendar.date(byAdding: .day, value: -7 * 5` syntax you used above,  maybe that, plus a constant thrown in for `overTwoMonthsAgo` might help me. Reading it again to see what is the intention of the test...

Reply:
Yeah, I agree it's not the clearest of tests. 10 weeks ago?

I didn't change the test — as the test wasn't failing — just moved it to a `XCTAssertEqual` so we could see the value when the test fails.

Reply:
ok no prob, good to know I wasn't the only one headscratching on this test